ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/Tables/CVEsTable.tsx

```diff
@@ -166,7 +166,7 @@ function CVEsTable({
                         Top CVSS
                     </TooltipTh>
                     <TooltipTh
-                        sort={getSortParams('Image sha', aggregateByDistinctCount)}
+                        sort={getSortParams('Image Sha', aggregateByDistinctCount)}
```

Comment:
Off the main subject of this contribution, what are pro and con of frontend counterpart to source of truth:
https://github.com/stackrox/stackrox/blob/master/pkg/search/options.go

Somewhat like feature flags:
https://github.com/stackrox/stackrox/blob/master/pkg/features/list.go
https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/src/types/featureFlag.ts

For type checking here:
https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/src/hooks/useFeatureFlags.ts#L26

Reply:
Going further out on that limb, explicit subset of which sort fields can participate in multiple sorting?

Reply:
I don't think that is a bad idea - ideally we could generate the code using the backend as the source of truth versus manual duplication though.

Good question on which fields can participate in multiple sorting -- I assume any but honestly am not sure. It would be good to test this separately and see if this is a general enhancement of the API or specific to just a subset. If the latter, it might make sense to build some checks into the code to make sure the fields are used appropriately.